### ISSUE TITLE
Make ALERT_TRIGGERED_OR_RESOLVED placeholder values configurable

### ISSUE THREAD

**Issue by GeorgFleig** [TwinProduction/gatus#76]
We're currently using Gatus for triggering health checks and having an internal dashboard. In order to unify with our public status reporting we'd like to use Gatus to set the status of components on our statuspage.io status page.

I've configured a custom alerting provider which works great when a service goes down. But I don't see a way to report it as up again at the moment. Statuspage.io requires different strings than `TRIGGERED` and `RESOLVED` to distinguish the statuses. Currently they are hard coded here `ALERT_TRIGGERED_OR_RESOLVED`: 
https://github.com/TwinProduction/gatus/blob/master/alerting/provider/custom/custom.go#L48

The configuration is as follows. I use the `ALERT_DESCRIPTION` to identify the service name at statuspage.io.
```
alerting:
  custom:
    url: "https://api.statuspage.io/v1/pages/XXX/components/[ALERT_DESCRIPTION]"
    method: "PATCH"
    headers:
      Authorization: "OAuth YYY"
    body: |
      { "component": { "status": "operational" } }

services:
  - name: foo
    url: "https://this.is.down"
    interval: 1m
    conditions:
      - "[STATUS] == 200"
    alerts:
      - type: custom
        enabled: true
        send-on-resolved: true
        description: a9gbjb11q8gy
```

Status of a component could be any of

```
"operational" "under_maintenance" "degraded_performance" "partial_outage" "major_outage" ""
```

(see https://developer.statuspage.io/#operation/patchPagesPageIdComponentsComponentId).

It would be great if the two strings were configurable on `alerting.custom` level, making the custom alert feature even more flexible. Maybe like this, defaulting to the current hardcoded-values:
```
alerting:
  custom:
    url: "https://api.statuspage.io/v1/pages/XXX/components/[ALERT_DESCRIPTION]"
    method: "PATCH"
    triggered_message: "partial_outage"
    resolved_message: "operational"
    headers:
      Authorization: "OAuth YYY"
    body: |
      { "component": { "status": "[ALERT_TRIGGERED_OR_RESOLVED]" } }
```

Gatus is great and keeps getting better. I wanted to say thanks for providing this neat tool and actively maintaining it!

**Comment by TwinProduction** [comment]
Fascinating - so you're using Gatus to push data onto a different status page?
I was originally afraid people wouldn't get creative enough with the `custom` alerting provider, but it looks like my worries were unnecessary.

But yeah, this seems like a good feature to add. My only concern is how to expose these configurations.
Off the top of my head, I think something like this would be acceptable:
```yaml
alerting:
  custom:
    url: "https://api.statuspage.io/v1/pages/XXX/components/[ALERT_DESCRIPTION]"
    method: "PATCH"
    placeholders:                        <--
      ALERT_TRIGGERED_OR_RESOLVED:       <--
        triggered: "partial_outage"      <--
        resolved: "operational"          <--
    headers:
      Authorization: "OAuth YYY"
    body: |
      { "component": { "status": "[ALERT_TRIGGERED_OR_RESOLVED]" } }
```

**Comment by GeorgFleig** [comment]
Yes, as the two tools serve slightly different purposes, we want to leverage the combination of both. Gatus is able to do the actual health check and also monitor all internal services, the public status page is used for displaying some of the result to the public, incident communication and notification subscription.

Regarding your proposal for the configuration syntax: That is definitely better than my initial idea :+1: 

A question on a slightly different topic that just came up: If I understand the current configuration syntax of alerts correctly it is possible to use Slack and a custom alerting method at the same time but not two different custom alerts, correct? Just out of curiosity if we ever wanted to add a second custom alert.

**Comment by TwinProduction** [comment]
@GeorgFleig That's correct, Gatus currently only supports one of each alerting providers, so at most, you could have 5 providers configured (1 slack, 1 pagerduty, 1 mattermost, 1 twilio and 1 custom).

**Comment by TwinProduction** [comment]
FYI, I made some changes and the syntax is now:
```yaml
alerting:
  custom:
    placeholders:
      ALERT_TRIGGERED_OR_RESOLVED:
        TRIGGERED: "partial_outage"
        RESOLVED: "operational"
```

**Comment by TwinProduction** [comment]
Released in [v2.2.0](https://github.com/TwinProduction/gatus/releases/tag/v2.2.0)
